### PR TITLE
feat(json): support JSON.GET pretty print options

### PR DESCRIPTION
**Pull request by iko1**
Fixes: https://github.com/dragonflydb/dragonfly/issues/848

### CODE REVIEW

**Comment by iko1** [comment]
@romange, there is a way to exclude the patch file in this commit from the clang-format validator?

**Comment by romange** [comment]
Hey @iko1  thanks for fixing this issue!

Yes, you can check `.pre-commit-config.yaml` file and add `patches` directory to the exclude filter.

File: src/server/json_family_test.cc [hunk 1]
```diff
@@ -110,7 +110,9 @@ TEST_F(JsonFamilyTest, SetGetFromPhonebook) {
   EXPECT_EQ(resp, R"(["New York","NY","21 2nd Street","10021-3100"])");
 
   resp = Run({"JSON.GET", "json", "$.firstName", "$.age", "$.lastName"});
-  EXPECT_EQ(resp, R"({"$.age":[27],"$.firstName":["John"],"$.lastName":["Smith"]})");
+  EXPECT_EQ(
```
Comment (romange) [inline src/server/json_family_test.cc:113]: could you please add few tests covering those cases?

Comment (iko1) [reply src/server/json_family_test.cc]: I reverted the change that affected the "JSON.GET" command when formatting args are not supplied with it.
Can you elaborate on which tests are currently missing?

Comment (iko1) [reply src/server/json_family_test.cc]: I added a few tests based on the previous JSON.GET cases that cover the feature functionality.

**Review by romange — APPROVED** [review]

**Pull request merged**